arangod/VocBase/Methods/Databases.cpp

```diff
@@ -111,13 +113,24 @@ arangodb::Result Databases::info(TRI_vocbase_t* vocbase, VPackBuilder& result) {
       }
       result.add("path", VPackValue("none"));
       result.add("isSystem", VPackValue(name[0] == '_'));
+
+      //would this be ok?
+      //arangodb::addOneShardOptionsToOpenObject(result, vocbase->sharding(), vocbase->replicationFactor());
+
+      //copy from plan slice
+      auto one = arangodb::getOneShardOptions(name, value);
+      arangodb::addOneShardOptionsToOpenObject(result, one.first, one.second);
+
     }
   } else {
     VPackObjectBuilder b(&result);
     result.add("name", VPackValue(vocbase->name()));
     result.add("id", VPackValue(std::to_string(vocbase->id())));
     result.add("path", VPackValue(vocbase->path()));
     result.add("isSystem", VPackValue(vocbase->isSystem()));
+    arangodb::addOneShardOptionsToOpenObject(result, vocbase->sharding(), vocbase->replicationFactor());
```

Comment:
Can't we have one method such as `TRI_vocbase_t::toVelocyPack(VPackBuilder& b)` which does the serialization of a vocbase object to VelocyPack?